lib/scala/searcher/src/main/scala/org/enso/searcher/sql/Tables.scala

```diff
@@ -81,10 +82,11 @@ case class FileVersionRow(path: String, digest: Array[Byte])
 /** The type of a suggestion. */
 object SuggestionKind {
 
-  val ATOM: Byte     = 0
-  val METHOD: Byte   = 1
-  val FUNCTION: Byte = 2
-  val LOCAL: Byte    = 3
+  val MODULE: Byte   = 0
+  val ATOM: Byte     = 1
+  val METHOD: Byte   = 2
+  val FUNCTION: Byte = 3
+  val LOCAL: Byte    = 4
```

Comment:
Doesn't it make more sense to use a literal enum here?

Reply:
Unfortunately, there is no enum type in SQLite